### DISCUSSION TITLE
Transmitter not connecting

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#3451] [General]
My transmitter failed and had to replace transmitter, but it does not seem to connect after I enter the new transmitter ID.  Is there a resource where I can see the proper steps for this?  Thanks

**Comment by Navid200** [comment]
You have not mentioned what transmitter.

If it is G6, here it is: https://navid200.github.io/xDrip/docs/Starting-G6.html

If it is not G6, tell us what it is.

**Reply by GLoren1** [reply]
Thank you. I followed these, but the transmitter still shows not connected.  This transmitter is new but expired. Is it possible that the battery is dead because of age?

**Reply by Navid200** [reply]
It depends.  What are the two dates on the transmitter box?
Have you activated the transmitter?

**Reply by GLoren1** [reply]
It finally connected
